docs/protocol.md

```diff
@@ -332,10 +332,18 @@ Used to control fragmentation. Valid flags:
 | flag   | description
 | ------ | -----------
 | `0x01` | more fragments follow
+| `0x02` | is request streaming
 
 If the fragments flag isn't set, then this is the only/last frame for this
 message id.
 
+If the streaming flag isn't set, then this request has a fragmented body
+instead of a streaming body.
```

Comment:
Fragmentation and streaming are orthogonal.

“If the streaming flag is set, then streaming request timeout semantics (time to first response frame) apply instead of the non-streaming semantics (time to last response frame).”.